process/due-diligence-guidelines.md

```diff
@@ -0,0 +1,142 @@
+
+* make sure you're clear on the [TOC Principles](https://github.com/cncf/toc/blob/toc-principles/principles.md),
+  the [project proposal process](https://github.com/cncf/toc/blob/master/process/project_proposals.adoc),
+  the [graduation criteria](https://www.cncf.io/projects/graduation-criteria/)
+  and [desired cloud native properties](https://www.cncf.io/about/charter/) are.  The project sponsor (a member
+  of the TOC) should have assisted in crafting the proposal to explain why it's a good fit for the CNCF. If anything's
+  unclear to you, reach out to the project sponsor or, failing that, the TOC mailing list for advice.   
+* make sure you've read, in detail, the relevant [project proposal](https://github.com/cncf/toc/tree/master/proposals),
+  This will usually be in the form of an [open pull request](https://github.com/cncf/toc/pulls).
+  Consider holding off on commenting on the PR until you've completed the next three steps.
+* take a look at some [previous submissions](https://github.com/cncf/toc/pulls?utf8=%E2%9C%93&q=is%3Apr)
+  (both successful and unsuccessful) to help calibrate your expectations.
+* do as much reading up as you need to (and consult with experts in the specific field) in order to familiarize yourself with the technology
+  landscape in the immediate vicinity of the project (and don't only use the proposal and that project's documentation as a guide in this regard).
+* at this point you should have a very clear technical idea of what exactly the project actually does and does not do, roughly how it compares with and differs from 
+  similar projects in it's technology area, and/or a set of unanswered questions in those regards. 
+* go through the [graduation criteria](https://www.cncf.io/projects/graduation-criteria/) and for each item,
+  decide for yourself whether or not you have enough info to make a strong, informed call on that item.
+  * If so, write it down, with motivation.
+  * If not, jot down what information you feel you're missing.
+  * Also take note of what unanswered questions the community might have posted in the PR review that you consider
+    to be critically important.
+    
+### Some example questions that will ideally need clear answers
+
+Most of these should be covered in the project proposal document.  The
+due diligence exercise involves validating any claims made there,
+verifying adequate coverage of the topics, and possibly summarizing
+the detail where necessary.
+
+#### Technical
+
+* An architectural, design and feature overview should be available.
+  ([example](https://github.com/docker/notary/blob/master/docs/service_architecture.md),
+  [example](https://github.com/docker/notary/blob/master/docs/command_reference.md))
+* What are the primary target cloud-native use cases?  Which of those:
+  * Can be accomplished now.
+  * Can be accomplished with reasonable additional effort (and are ideally already on the project roadmap).
+  * Are beyond the current roadmap.
```

Comment:
Just calling these cases out as part of the DD is fine. The TOC can decide the consequences.

I think it is reasonable for some use cases to be out of scope. Engineering tradeoffs need to be made. We may seek multiple projects providing similar functionality when they make distinctly different design tradeoffs.

Reply:
Gotcha.  Added some additional clarification.